agent/dockerclient/versions.go

```diff
@@ -30,6 +30,8 @@ const (
 	Version_1_28 DockerVersion = "1.28"
 	Version_1_29 DockerVersion = "1.29"
 	Version_1_30 DockerVersion = "1.30"
+	Version_1_31 DockerVersion = "1.31"
```

Comment:
Why are we adding these versions?

Reply:
Fsouza supports up to apiVersion v1.35, which was added for volumes support.